### PR TITLE
2.4 release version bump

### PR DESCRIPTION
**Pull request by jfy133**
In preparation for the next release:

- Version bump
- Disable sending both raw and refined bins to downstream steps due to file name collisions, to be fixed soon in #489
- Fixes path for picking up MaxBin2 bins 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by d4straub — APPROVED** [review]
LGTM

File: nextflow.config [hunk 1]
```diff
@@ -371,7 +371,7 @@ manifest {
     description     = """Assembly, binning and annotation of metagenomes"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '2.4.0dev'
+    version = '2.4.0'
```
Comment (FriederikeHanssen) [inline nextflow.config:374]: this is an harshil unworthy indent

**Review by FriederikeHanssen — APPROVED** [review]

File: nextflow.config (hunk 1 above)
Comment (jfy133) [reply nextflow.config:374]: Don't blame me! Blame tools!!!!

Comment (d4straub) [reply nextflow.config:374]: Yes, made by tools & requires manual correction. I gave up on that :D

**Pull request merged**